packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -196,6 +199,7 @@ export abstract class FluidDataStoreContext extends EventEmitter implements
         public readonly isLocalDataStore: boolean,
         bindChannel: (channel: IFluidDataStoreChannel) => void,
         protected pkg?: readonly string[],
+        protected isRootDataStore?: boolean,
```

Comment:
Does it have to be optional?

Reply:
I believe you can get rid of this argument altogether from FluidDataStoreContext, and have it only in LocalFluidDataStoreContextBase. All the other usages can use local variable on the stack

Reply:
Removed it from FluidDataStoreContext.